saleor/graphql/attribute/utils.py

```diff
@@ -52,6 +52,8 @@ class AttrValuesInput:
 T_INPUT_MAP = List[Tuple[attribute_models.Attribute, AttrValuesInput]]
 T_ERROR_DICT = Dict[Tuple[str, str], List[str]]
 
+EntityTypeData = namedtuple("EntityTypeData", ["model", "entity_field", "value_field"])
```

Comment:
Minor improvement - could we rename `entity_field` to something like `name_field`? If I understand correctly, it represents a field that for each type should be used to get the reference's name, right?

Reply:
Yep. I'll change it